README.md

```diff
@@ -1,2 +1,350 @@
+- The provided DID string MUST be compliant with the did:itn method.
+- The provided DID document MUST be compliant with the DID V1.0 DID document specification.
+
+---
+
+#### Revoke
+
+Description: Based on the DID data schema context file and DID method an ITN process revokes an existing DID and its DID document in accordance with the W3C DID standard.
+
+Requirements:
+
+- `did` input parameter MUST be a DID string that has to be revoked.
+- `receiver` input parameter can be a DID string or a DID Document.
+- The provided DID string MUST be compliant with the did:itn method.
+- The provided DID document MUST be compliant with the DID V1.0 DID document specification.
+
+---
+
+#### Recover
+
+Description: Based on the DID data schema context file and DID method, an ITN process recovers a DID and its DID document in accordance with the W3C DID standard.
+
+Requirements:
+
+- `did` input parameter MUST be a DID string that has to be recovered.
+- `receiver` input parameter can be a DID string or a DID Document.
+- The provided DID string MUST be compliant with the did:itn method.
+- The provided DID document MUST be compliant with the DID V1.0 DID document specification.
+
+---
+
+#### Deactivate
+
+The `revoke` operation also deactivate the DID.
+
+## Sample DID Document
+
+```json
+{
+  "didDocument": {
+    "@context": [
+      "https://www.w3.org/ns/did/v1"
+    ],
+    "assertionMethod": [
+      "did:itn:NL28Bubaa8xUmhJAARNkNS#z6MkiYmb7ncfDAAhcxtAAGsm1oySq4KYJtsi2wHaoKAnmVXC"
+    ],
+    "authentication": [
+      "did:itn:NL28Bubaa8xUmhJAARNkNS#z6MkiYmb7ncfDAAhcxtAAGsm1oySq4KYJtsi2wHaoKAnmVXC"
+    ],
+    "id": "did:itn:NL28Bubaa8xUmhJAARNkNS",
+    "keyAgreement": [
+      {
+        "id": "did:itn:NL28Bubaa8xUmhJAARNkNS#z6LSd76Tk2UJ3RVxcTjYjZ8TfvHMQgZG5HmbKPmCidPXyTGZ",
+        "type": "X25519KeyAgreementKey2020",
+        "controller": "did:itn:NL28Bubaa8xUmhJAARNkNS",
+        "publicKeyMultibase": "z6LSd76Tk2UJ3RVxcTjYjZ8TfvHMQgZG5HmbKPmCidPXyTGZ"
+      }
+    ],
+    "verificationMethod": [
+      {
+        "id": "did:itn:NL28Bubaa8xUmhJAARNkNS#z6MkiYmb7ncfDAAhcxtAAGsm1oySq4KYJtsi2wHaoKAnmVXC",
+        "type": "Ed25519VerificationKey2020",
+        "controller": "did:itn:NL28Bubaa8xUmhJAARNkNS",
+        "publicKeyMultibase": "z6MkiYmb7ncfDAAhcxtAAGsm1oySq4KYJtsi2wHaoKAnmVXC"
+      }
+    ]
+  },
+  "didDocumentMetadata": {
+    "created": "2023-11-10T15:20:59Z",
+    "updated": "2023-11-10T15:20:59Z",
+    "deactivated": false,
+    "recoveryCommitment": "99cdefca761b58664641d3f0b89b4f1e04ad833b7a24a726e3424bc6c17f39a0"
+  },
+  "didResolutionMetadata": { }
+}
+```
+
+## Security Considerations
+
+For all `did:itn` DIDs, the initial asset creation, and subsequent updates are executed using `Ed25519` keys, which are widely recognized as a robust and secure cryptographic mechanism. `Confirm With Umed & Andreas - if the keys are different. DELETE THIS COMMENT LATER.`
```

Comment:
@pleerock can you confirm if this is accurate or if anything needs to change? cc: @Therecanbeonlyone1969